libs/sync/src/index.ts

```diff
@@ -0,0 +1,31 @@
+export { QuestionsComponent } from './lib/components/questions/questions.component';
+export { RegistrationComponent } from './lib/components/registration/registration.component';
```

Comment:
seems like this hasn't been marked as moved and the history get lost. 
See if there's any way to recover it, but no worries if it's too hard.

Reply:
Yeah, the history gets lost, I think it won't be possible to mark as moved